core/src/main/java/org/apache/accumulo/core/spi/compaction/DefaultCompactionPlanner.java

```diff
@@ -167,6 +167,12 @@ public String toString() {
       justification = "Field is written by Gson")
   @Override
   public void init(InitParameters params) {
+
+    if (params.getOptions().containsKey("executors")
+        && params.getOptions().get("executors").isBlank()) {
+      throw new IllegalStateException("No defined executors for this planner");
+    }
+
```

Comment:
What happens in the next statement if `params.getOptions()` does not contain `executors`?

Reply:
Added a test case for that behavior and found that an NPE would still be thrown. 
Modified the logic to handle that case as well.